olexandr.vakulinskiy/HW2/Task1/tamagotchi.rb

```diff
@@ -0,0 +1,111 @@
+class Tamagotchi
+
```

Comment:
Good, you used empty lines after attribute accessor, but she doesn't have to be between the name and accessor